lib/src/Uncertainty/Algorithm/MetaModel/FunctionalChaos/openturns/FunctionalChaosResult.hxx

```diff
@@ -100,6 +103,15 @@ public:
   /** Composed meta model accessor */
   virtual Function getComposedMetaModel() const;
 
+  /** Residuals accessor */
+  virtual Sample getSampleResiduals() const;
+
+  /** isLeastSquares_ accessor */
+  virtual Bool getIsLeastSquares() const;
+
+  /** isModelSelection accessor */
+  virtual Bool getIsModelSelection() const;
```

Comment:
rename it into `isModelSelection()` as you check a property

Reply:
This is fixed at 8d8ebe32519c39c30f3989424581f6336673145e. You can check it at [FunctionalChaosResult.hxx#L111](https://github.com/mbaudin47/openturns/blob/d2d18e244254b79129c880b1857a7c416ff819f3/lib/src/Uncertainty/Algorithm/MetaModel/FunctionalChaos/openturns/FunctionalChaosResult.hxx#L111)